activities/MindMath.activity/js/activity.js

```diff
@@ -289,16 +292,6 @@ const app = Vue.createApp({
 
     },
 
-    localized: function() {
-      this.$refs.SugarTutorial.activityLocalized(this.SugarL10n.localize);
-      document.getElementById('next-slot-text').innerHTML = this.SugarL10n.get("NextSlot");
-      document.getElementById('no-timer-button').innerHTML = this.SugarL10n.get("NoTimer");
-      document.getElementById('easy-button').title = this.SugarL10n.get("EasyLevel");
-      document.getElementById('medium-button').title = this.SugarL10n.get("MediumLevel");
```

Comment:
We don't to handle palette strings here as it'll be handled in their respective palette.js file
Hence we also don't have to wait for activity initialization(i.e all the palette are on the DOM to be accessed via document.getEle...)  to trigger "localized" event.
This code: https://github.com/llaske/sugarizer/pull/1381/commits/625c05f3a2770cab07d7c98aee77450ca0ae138a#diff-a6d6d8775bb10d9a02bd716602f205816dd159dcd42e01c89ea6a081a367dcdeL21-L33